components/x11/libxkbfile/Makefile

```diff
@@ -12,40 +12,30 @@
 #
 # Copyright 2016 Aurelien Larcher
 #
+BUILD_BITS=64_and_32
 
 include ../../../make-rules/shared-macros.mk
+include $(WS_MAKE_RULES)/x11.mk
 
 COMPONENT_NAME=           libxkbfile
-COMPONENT_VERSION=        1.0.9
+COMPONENT_VERSION=        1.1.0
```

Comment:
Are there any incompatibilities?

Reply:
I did not find anything suspicious: https://cgit.freedesktop.org/xorg/lib/libxkbfile/log/.